avocado_vt/plugins/vt_joblock.py

```diff
@@ -52,10 +69,9 @@ def _set_lock(self, job):
         try:
             with open(path, 'w') as lockfile:
                 lockfile.write("%u" % os.getpid())
-            self.lock_file = path
-        except IOError as e:
-            msg = ('Failed to create VT Job lock file "%s". Exiting...' % path)
-            self._abort(msg, job)
+            return path
+        except Exception as e:
```

Comment:
@clebergnu 
Hi, thanks, I will check this soon hopefully when I finish building my test environments